integration-cli/docker_cli_run_test.go

```diff
@@ -1038,7 +1038,7 @@ func (s *DockerSuite) TestRunUnprivilegedCannotMount(c *check.C) {
 
 func (s *DockerSuite) TestRunSysNotWritableInNonPrivilegedContainers(c *check.C) {
 	// Not applicable for Windows as there is no concept of unprivileged
-	testRequires(c, DaemonIsLinux)
+	testRequires(c, DaemonIsLinux, NotArm)
```

Comment:
Running the test manually on a Scaleway machine
```
root@scw-04:~# docker run armhf/busybox touch /sys/kernel/profiling
touch: /sys/kernel/profiling: Read-only file system
root@scw-04:~# docker run --privileged armhf/busybox touch /sys/kernel/profiling
touch: /sys/kernel/profiling: Permission denied
```
What are the best practices to investigate such issues?